mlkem/indcpa.h

```diff
@@ -20,6 +20,70 @@
 #include "common.h"
 #include "poly_k.h"
 
+#define mlk_indcpa_secret_key MLK_NAMESPACE_K(mlk_indcpa_secret_key)
+typedef struct
+{
+  mlk_polyvec skpv;
```

Comment:
Have you considered caching the mulcache? This would be useful in decapsulation.